### PR TITLE
#2232 Fix node-sass deprecation warning

### PR DESCRIPTION
**Pull request by desfero**

### CODE REVIEW

File: app/styles/global.scss [hunk 1]
```diff
@@ -1,5 +1,4 @@
 @import "~react-id-swiper/src/styles/scss/swiper";
-@import "~react-datetime/css/react-datetime";
```
Comment (desfero) [inline app/styles/global.scss]: moved for consistency to the same place where component is used

File: package.json [hunk 2]
```diff
@@ -143,7 +143,7 @@
     "mini-css-extract-plugin": "^0.4.1",
     "mocha": "^4.1.0",
     "mockdate": "^2.0.2",
-    "node-sass": "^4.9.3",
+    "node-sass": "^4.11.0",
```
Comment (desfero) [inline package.json]: a simple update was enough to fix warnings. 

From node-sass changelog
- Removes noisey deprecation warning for @import'ing .css files

**Comment by desfero** [comment]
visual regression ok right now

File: package.json (hunk 2 above)
Comment (Moejoe90) [reply package.json]: :+1:

**Pull request merged**